gcovr/writer/html/__init__.py

```diff
@@ -67,15 +67,20 @@ def __call__(self, *args):
         return self.get(*args)
 
 
-# Loading Jinja and preparing the environmen is fairly costly.
+# Loading Jinja and preparing the environment is fairly costly.
 # Only do this work if templates are actually used.
 # This speeds up text and XML output.
 @Lazy
-def templates():
-    from jinja2 import Environment, PackageLoader
+def templates(options):
+    from jinja2 import Environment, PackageLoader, FileSystemLoader
+
+    loader = PackageLoader("gcovr.writer.html")
+
+    if options.template_dir is not None:
+        loader = FileSystemLoader(options.template_dir)
 
     return Environment(
-        loader=PackageLoader("gcovr.writer.html"),
+        loader=loader,
```

Comment:
Is there also a list of loaders possible? Then the user can override a specific template but reusing the others.

Reply:
Yeah that's a good idea, Jinja2 does have a `ChoiceLoader` which I just added into the mix. This way a user only has to override the templates they care to change and don't have to copy them all into a directory.

Reply:
Can you also update the docs here?